lib/controllers/services.ts

```diff
@@ -123,3 +148,46 @@ export function GetService(serviceModel: ServiceModel): express.RequestHandler {
     )
   );
 }
+
+/**
+ * Returns the serviceId for all the Services that have sent
+ * at least one notification to the recipient with the provided fiscalCode.
+ */
+export function GetSenderServicesHandler(
```

Comment:
I would align the method name to the operationId of the API spec (`getServicesByRecipient`)